app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,75 @@
+module TreeNode
+  module Menu
+    class Item < Node
+      set_attribute(:key) { "#{@options[:node_id_prefix]}__#{@object.feature}" }
```

Comment:
Legacy. I still need to look into the keys that the controller expects to make updating work.